pkg/tfshim/tfplugin5/provider.go

```diff
@@ -403,7 +403,7 @@ func (p *provider) Apply(
 		return nil, err
 	}
 
-	resp, err := p.client.ApplyResourceChange(context.TODO(), &proto.ApplyResourceChange_Request{
+	resp, err := p.client.ApplyResourceChange(ctx, &proto.ApplyResourceChange_Request{
```

Comment:
Appreciate you cleaning up here but these seem unrelated to the PR, should we separate them?

Reply:
Yeah. I removed them from this PR and I'll open a new PR.